docs/layouts/partials/navbar-version-selector.html

```diff
@@ -0,0 +1,8 @@
+<a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
```

Comment:
I overrode the default version of this partial so that it's structured similarly to the other non-drop down menus with the li > a > span structure, this way it can inherit styles just like the rest of the menu items and not be a totally one-off case.